rom/laghos_rom.cpp

```diff
@@ -3177,9 +3177,34 @@ void ROM_Operator::ComputeReducedMv()
                 (*Wmat)(i,j) = Mvj[i];
         }
     }
+	// TODO: do I need to enforce MPI rank == 0?
```

Comment:
Note my question here: do I need to do the computation of the reduced Mv inverse sequentially on one process only, or can it take place in parallel? For the basic EQP case is done in parallel, so I left it like that for this case as well.